doc/migrations/conditions.md

```diff
@@ -0,0 +1,44 @@
+# New condition syntax 
+
+With `limitador-server` version `1.0.0` (and the `limitador` crate version `0.3.0`), the syntax for `condition`s within 
+`limit` definitions has changed.
+
+## Changes
+
+### The new syntax
+
+The new syntax formalizes what part of an expression is the identifier and which is the value to test against. 
+Identifiers are simple string value, while string literals are to be demarcated by single quotes (`'`) or double quotes
+(`"`) so that `foo == " bar"` now makes it explicit that the value is to be prefixed with a space character.
+
+A few remarks:
+ - Only `string` values are supported, as that's what they really are
+ - There is no escape character sequence supported in string literals
+ - A new operator has been added, `!=`
```

Comment:
Does this conflict with this other statement from the **How it works** page?

https://github.com/Kuadrant/limitador/blob/1576eeb174029c5eb22bb4f06763f5211a1887b0/doc/how-it-works.md?plain=1#L40